SDK_CHANGELOG.md

```diff
@@ -5,6 +5,13 @@
 
 ## Releases
 
+### v0.35.0 - Tech Preview (12/24/2018)
```

Comment:
Version here was changed to 0.35.0 but not in the SdkVersion.Minor value in the api.proto file. You have to change it there also.

Reply:
You might as well take the opportunity to change the date on the Changelog